SpywareFilter/sections/allowlist.txt

```diff
@@ -420,10 +420,17 @@ login.coupang.com#%#//scriptlet('prevent-element-src-loading', 'script', 'app-co
 ! https://make.dmm.com/mypage/upload/ broken upload
 ||googletagmanager.com/gtm.js$script,redirect=googletagmanager-gtm,domain=make.dmm.com,important
 @@||googletagmanager.com/gtm.js$domain=make.dmm.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/168620
+! https://github.com/AdguardTeam/AdguardFilters/issues/168622
 ! https://github.com/AdguardTeam/AdguardFilters/issues/152345
-capital.it#%#//scriptlet('prevent-element-src-loading', 'script', 'static.chartbeat.com/js/chartbeat_brightcove_plugin.js')
+m2o.it,deejay.it,capital.it#%#//scriptlet('prevent-element-src-loading', 'script', 'static.chartbeat.com/js/chartbeat_brightcove_plugin.js')
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
-@@||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$domain=capital.it
+@@||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$domain=m2o.it|deejay.it|capital.it
+!+ PLATFORM(ext_ublock)
```

Comment:
Hint is applied only to one rule. Better move a rule without hint above it. If both required only for uBO - use `if-endif` directive.

Reply:
I'll explain.

As regards chartbeat_brightcove_plugin
1) `#%#//scriptlet('prevent-element-src-loading', 'script', 'static.chartbeat.com/js/chartbeat_brightcove_plugin.js')`
is only for AdGuard (except content blockers)
2) `@@||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$domain=m2o.it|deejay.it|capital.it`
is only for AdGuard content blockers
3) `||static.chartbeat.com/js/chartbeat_brightcove_plugin.js$script,redirect=noopjs,important,domain=m2o.it|deejay.it|capital.it`
is only for uBO, because it doesn't support `prevent-element-src-loading` scriptlet. If you prefer it can be used `redirect=noopjs` for both Adguard (except content blockers) and uBO.

As regards googletagmanager
1) `||googletagmanager.com/gtm.js$script,redirect=google-analytics,important,domain=m2o.it|deejay.it|capital.it` is for both Adguard (except content blockers) and uBO;
2) `@@||googletagmanager.com/gtm.js$domain=m2o.it|deejay.it|capital.it` is only for AdGuard content blockers

Reply:
@Alex-302 
Since AnnoyancesFilter/Cookies/sections/cookies_specific.txt conflicts, if you prefer I can create a new (updated) branch and send a new pull request.
Let me know how to manage SpywareFilter/sections/allowlist.txt